src/phoenix/experimental/evals/templates/default_templates.py

```diff
@@ -97,15 +97,15 @@
     [END DATA]
     Compare the Summary above to the Original Document and determine if the Summary is
     comprehensive, concise, coherent, and independent relative to the Original Document.
-    Your response must be a string, either Good or Bad, and should not contain any text
-    or characters aside from that. Bad means that the Summary is not comprehensive, concise,
-    coherent, and independent relative to the Original Document. Good means the Summary
+    Your response must be a string, either good or bad, and should not contain any text
+    or characters aside from that. The string bad means that the Summary is not comprehensive, concise,
+    coherent, and independent relative to the Original Document. The string good means the Summary
```

Comment:
maybe add some quotes to make it stand out more (at least for human eyes)
```suggestion
    Your response must be a string, either "good" or "bad", and should not contain any text
    or characters aside from that. The string "bad" means that the Summary is not comprehensive, concise,
    coherent, and independent relative to the Original Document. The string "good" means the Summary
```

Reply:
yeah i had that originally, but i'm afraid, the LLM might include them in the output. Happy to do either. That would also cause the bug with that notebook. What do you think @axiomofjoy

Reply:
good point. fortunately quotes in the response are stripped in or processing

Reply:
ahh okay, even better. Still learning the library

Reply:
Thanks for the commit!